requirements/run.in

```diff
@@ -1 +1,4 @@
 requests==2.27.0
+
```

Comment:
@rmotitsuki could you please remove `urllib3` and `charset-normalizer` transitive dependencies from this run.in file? It turns out that in the `requirements/run.in` we only pin the main library that we depend on, and then the transitive ones are pineed on `requirements/run.txt` (that way we always can rely on `in.txt` and know for sure which are the main ones). 

If you check the [`requirements/run.txt`](https://github.com/kytos-ng/coloring/blob/master/requirements/run.txt) file we've already got these dependencies there, I think you should update this line on [`setup.py`](https://github.com/kytos-ng/coloring/blob/master/setup.py#L214) and pass the `requirements/run.txt` instead of the `requirements/in.txt`, could you please update that part as well, since you're at it? and then for completeness you an generated the dev dependencies on python 3.9 again like you've done.

Reply:
FYI, as a side effect for instance, as it is, it would break AmLight's docker build, since we're freezing and maintaining all run.txt and run.in with same version across all NApps (`requests==2.27.0
`,  `urllib3==1.26.7`,  `certifi==2021.10.8`, `charset-normalizer==2.0.10`, `certifi==2021.10.8`):

```
Installing collected packages: urllib3, charset-normalizer, dill, amlight-sdntrace Attempting uninstall: urllib3
    Found existing installation: urllib3 1.26.9
ERROR: Cannot uninstall urllib3 1.26.9, RECORD file not found. You might be able to recover from this via: 'pip install --force-reinstall --no-deps urllib3==1.26.9'.
WARNING: You are using pip version 21.3.1; however, version 22.0.4 is available.
You should consider upgrading via the '/usr/bin/python3 -m pip install --upgrade pip' command.
The command '/bin/sh -c python3 -m pip install -e git+https://github.com/kytos-ng/storehouse@${branch_storehouse}#egg=kytos-storehouse  && python3 -m pip install -e git+https://github.c
```